backend/config/routes.rb

```diff
@@ -1,4 +1,4 @@
-Spree::Core::Engine.routes.draw do
+Spree::Core::Engine.add_routes do
```

Comment:
In the meantime, this [got deprecated](https://github.com/solidusio/solidus/blob/f831961f0c55dfcfeaf72bbb0804e7e325a6c671/core/lib/spree/core/routes.rb#L5)